### PR TITLE
west.yml: Update hal_stm32 with recent cube packages

### PR DESCRIPTION
**Pull request by FRASTM**
STM32Cube updates:
    stm32cube: update stm32l5 to version V1.4.0
    stm32cube: update stm32l4 to version V1.17.0
    stm32cube: update stm32g4 to version V1.4.0
    stm32cube: update stm32f7 to version V1.16.1
    stm32cube: update stm32f4 to version V1.26.0
    stm32cube: update stm32g0 to version V1.4.1
    stm32cube: update stm32h7 to version V1.9.1
    stm32cube: update stm32mp1 to version V1.4.0

    update stm32cube/common_ll
    
    Signed-off-by: Francois Ramu <francois.ramu@st.com>

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -72,7 +72,7 @@ manifest:
       revision: b52fdbf4b62439be9fab9bb4bae9690a42d2fb14
       path: modules/hal/st
     - name: hal_stm32
-      revision: 50283e0438d882a80fb681907664491c7042c817
+      revision: a8498d3aa79595a3ca51ad2dd341d9f28d8343c0
```
Comment (erwango) [inline west.yml:75]:
```suggestion
      revision: pull/93/head
```
The interest of this method is to be sure we point on and testing the last PR version.

**Review by erwango — CHANGES_REQUESTED** [review]

File: west.yml (hunk 1 above)
Comment (FRASTM) [reply west.yml]: OK, changed to `pull/93/head`

**Comment by FRASTM** [comment]
compilation ok with :
 ```
   west build -p auto -b nucleo_g071rb samples/hello_world/
   west build -p auto -b nucleo_f746zg samples/hello_world/
   west build -p auto -b nucleo_h743zi samples/hello_world/
   west build -p auto -b nucleo_f446re samples/hello_world/
   west build -p auto -b nucleo_g474re samples/hello_world/
   west build -p auto -b nucleo_l4r5zi samples/hello_world/
   west build -p auto -b nucleo_l552ze_q samples/hello_world/

```

**Comment by FRASTM** [comment]
rebase

**Comment by erwango** [comment]
@FRASTM https://github.com/zephyrproject-rtos/hal_stm32/pull/93 has been merged. You can update this PR.

**Comment by FRASTM** [comment]
rebase and update as the  zephyrproject-rtos /hal_stm32 is now merged with latest versions of the STM32Cube

**Review by erwango — APPROVED** [review]

**Review by ABOSTM — APPROVED** [review]

**Pull request merged**